validphys2/src/validphys/config.py

```diff
@@ -1286,17 +1286,16 @@ def produce_rules(
         default_filter_rules=None,
         filter_rules=None,
         default_filter_rules_recorded_spec_=None,
-        added_filter_rules: (list, type(None)) = None,
+        added_filter_rules: (tuple, type(None)) = None,
     ):
         """Produce filter rules based on the user defined input and defaults."""
-        from validphys.filters import Rule, RuleProcessingError, default_filter_rules_input
 
         theory_parameters = theoryid.get_description()
 
         if filter_rules is None:
             # Don't bother loading the rules if we are not using them.
             if use_cuts is not CutsPolicy.INTERNAL:
-                return None
+                return tuple()
```

Comment:
Is this change necessary? `None` should be hashable and this could have collateral effects (like some code doing `if x is None`)

Reply:
I had some trouble at some point with a bit of the code saying `non Iterable` and therefore changed it to `tuple()`

I now changed it back, let's see after the tests